src/main/resources/application.properties

```diff
@@ -4,6 +4,11 @@ workspace.maxStairwayThreads=4
 workspace.resourceId=mc-terra-workspace-manager
 workspace.stairwayTimeoutSeconds=1800
 
+stackdriver.serviceAccountFilePath=src/test/resources/rendered/wsm-firecloud-account.json
```

Comment:
I think we probably want a way to enable/disable tracing via configuration. As is, you can't run WM at all without access to a firecloud account, which is a problem for HCLS/anyone else external.

Relying on stuff in the test directory for our main workflow worries me a bit. Could this be rendered to the main resources directory?

Reply:
I changed it so that this file renders in main. Also created [AS-455](https://broadworkbench.atlassian.net/browse/AS-455) to re-think WSM service accounts. The current SA we're using isn't dedicated to WSM. 

As for enabling/disabling tracing - I can add that to config and do something like:

```
if (traceEnabled) {
    createAndRegister();
}
```

This will mean that traces are created, but will not be exported. There might be better ways to do this though. The alternative way I found (https://github.com/DataBiosphere/terra-workspace-manager/pull/61) has an enable/disable feature